src/libYARP_sig/src/Image.cpp

```diff
@@ -657,7 +686,6 @@ void Image::resize(int imgWidth, int imgHeight) {
     yAssert(imgWidth>=0 && imgHeight>=0);
 
     int code = getPixelCode();
-    int size = getPixelSize();
     bool change = false;
     if (code!=imgPixelCode) {
         imgPixelCode = code;
```

Comment:
I think you should be setting `imgPixelSize` as well here

Reply:
again here i think it's better use the method (that is linked to the pixelSize property) for making clear the binding between the two. (someone who read this, by going to the implementation of getPixelSize() sees immediatly the binding)